test/e2e/pods.go

```diff
@@ -693,7 +693,8 @@ var _ = Describe("Pods", func() {
 									Port: intstr.FromInt(8080),
 								},
 							},
-							InitialDelaySeconds: 30,
+							InitialDelaySeconds: 15,
+							TimeoutSeconds:      30,
```

Comment:
Bumping from 1s (default) to 30s seems a little bit extreme. Is there any reason to go so far and will the test timeout be able to accommodate this?